packages/shared/src/components/cards/FeedbackCard.tsx

```diff
@@ -55,7 +32,7 @@ export const FeedbackCard = ({ post }: FeedbackCardProps): ReactElement => {
         <Button
           id="upvote-post-btn"
           pressed={post?.userState?.vote === UserPostVote.Up}
-          onClick={() => toggleUpvote(post)}
+          onClick={() => toggleUpvote({ post, origin: Origin.Feed })}
```

Comment:
This can get rendered on feed and modal right?
Not sure the origin matches in that case? (But now wondering how it was before then)

Reply:
Feedback card is only for cards on feed. This is the new component that was added for engagement loops.